package.json

```diff
@@ -29,7 +29,7 @@
     "react": "17.0.2",
     "react-dom": "17.0.2",
     "react-query": "^3.39.1",
-    "react-redux": "7.2.4",
+    "react-redux": "8.0.2",
```

Comment:
@amirfefer should we remove redux? :thinking:

Reply:
we need it for trigger notifications :(